bin/v-update-letsencrypt-ssl

```diff
@@ -102,6 +113,13 @@ for user in $($HESTIA/bin/v-list-sys-users plain); do
             else
                 $BIN/v-log-action "$user" "Info" "Web" "Let's Encrypt SSL certificate renewed (Domain: $domain)."
             fi
+            if [[ "$domain_forcessl" == 'yes' ]] ; then
+                $BIN/v-add-web-domain-ssl-force $user $domain
+            fi
+            if [[ "$domain_redirect" != '' ]] ; then
+                $BIN/v-add-web-domain-redirect $user $domain $domain_redirect $domain_redirect_code
+            fi
+
         fi
     done
 
```

Comment:
It probally doesn't hurt to add a restart on the end as both v-add-web-domain-ssl-force and v-add-web-domain-redirect don't restart the web server on default